source/common/network/address_impl.cc

```diff
@@ -173,10 +174,9 @@ std::string Ipv4Instance::sockaddrToString(const sockaddr_in& addr) {
 }
 
 absl::uint128 Ipv6Instance::Ipv6Helper::address() const {
-  absl::uint128 result{0};
+  absl::uint128 result;
   static_assert(sizeof(absl::uint128) == 16, "The size of asbl::uint128 is not 16.");
-  memcpy(static_cast<void*>(&result), static_cast<const void*>(&address_.sin6_addr.s6_addr),
-         sizeof(absl::uint128));
+  safe_memcpy(&result, reinterpret_cast<const absl::uint128*>(&address_.sin6_addr.s6_addr));
```

Comment:
In this case the issue is the type of ```result``` (absl::uint128), so I guess it would be better to create a constructor that allowed ```const uint8_t[16]``` as parameter and initialise ```result``` with an assigment.

The error is:
```
source/common/network/address_impl.cc:179:50:   required from here
bazel-out/k8-fastbuild/bin/source/common/common/_virtual_includes/safe_memcpy_lib/common/common/safe_memcpy.h:12:9: error: 'void* memcpy(void*, const void*, size_t)' copying an object of type 'class absl::uint128' with 'private' member 'absl::uint128::lo_' from an array of 'const unsigned char [16]'; use assignment or copy-initialization instead [-Werror=class-memaccess]
   12 |   memcpy(dst, src, sizeof(T1));
      |   ~~~~~~^~~~~~~~~~~~~~~~~~~~~~
```

Reply:
Got it. OK I think what you ave here is good then.